backend/src/v5/schemas/tickets/validators.js

```diff
@@ -53,6 +53,13 @@ Validators.propTypesToValidator = {
 			forward: types.position.required(),
 			up: types.position.required(),
 			size: Yup.number().when('type', (type, schema) => (type === CameraType.ORTHOGRAPHIC ? schema.required() : schema.strip())),
+			clippingPlanes: Yup.array().of(
```

Comment:
Can we move this out of the camera

so it's at the same level as `screenshot`, `state` etc